src/pages/Candidate/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { Fragment, useEffect } from 'react';
+import { useHistory } from "react-router-dom";
+import PageTitle from '../../layout-components/PageTitle';
+import RegularTable from '../../components/RegularTable';
+import LabelText from '../../components/Label/LabelText'
+
+import {
+    Grid,
+    Card,
+    CardContent,
+    Tooltip,
+} from '@material-ui/core';
+
+import { useEvernode } from '../../services/Evernode';
+import Loader from '../../components/Loader';
+
+const Candidate = (props) => {
+    const history = useHistory();
+    const evernode = useEvernode();
+    const candidateId = props.match.params.candidateId;
+    const [info, setInfo] = React.useState(null);
+
+    useEffect(() => {
+        const fetchInfo = async () => {
+            setInfo(null);
+            const candidate = await evernode.getCandidateById(candidateId);
+            const candidateInfo = (candidate) ? candidate : null;
+            const tableHeadings = {
+                key: 'Key',
+                value: 'Value'
+            }
+            let tableValues = candidateInfo ? [
+                {
+                    key: 'Created Timestamp',
+                    value: <Tooltip title="Created Timestamp"><span>{candidateInfo.createdTimestamp
+                    }</span></Tooltip>
+                },
+                {
+                    key: 'Foundation Vote Status',
+                    value: <Tooltip title="Foundation Vote Status">
+                        <>
+                            {candidateInfo.foundationVoteStatus === "supported" ? <LabelText labelType="success">
+                                Supported
+                            </LabelText> : <LabelText labelType="warning">
+                                Rejected
+                            </LabelText>}
+                        </>
+                    </Tooltip>
+                },
+                {
+                    key: 'Last Vote Timestamp',
+                    value: <Tooltip title="Last Vote Timestamp">
+                        <span>{candidateInfo.lastVoteTimestamp}</span>
+                    </Tooltip>
+                },
+                {
+                    key: 'Owner Address',
+                    value: <Tooltip title="Owner Address">
+                        <span>{candidateInfo.ownerAddress}</span>
+                    </Tooltip>
+                },
+                {
```

Comment:
This will be 0 for piloted mode candidate, So tet's hide this if `!candidateInfo.proposalFee`